docker-versions-maven-plugin/src/main/java/com/github/cafapi/docker_versions/plugins/IgnoreVersion.java

```diff
@@ -76,4 +78,26 @@ public String toString()
     {
         return "IgnoreVersion [version=" + version + ", type=" + type + "]";
     }
+
+    @Override
+    public int hashCode()
+    {
+        return Objects.hash(type, version);
+    }
+
+    @Override
+    public boolean equals(final Object obj)
+    {
+        if (this == obj) {
+            return true;
+        }
+        if (obj == null) {
+            return false;
+        }
+        if (getClass() != obj.getClass()) {
```

Comment:
This is ok but given that the class is `final` you could have used `instanceof`.